applications/views.py

```diff
@@ -218,6 +221,22 @@ def post(self, request, *args, **kwargs):
             status=status.HTTP_200_OK,
         )
 
+    def validate_linkedin_url(self, linkedin_url):
+        """
+        Validate that a LinkedIn URL has the right format and length
+
+        Args:
+            linkedin_url (string): LinkedIn URL of a user
+        """
+        if len(linkedin_url) > 200:
+            raise ValidationError(
+                {"errors": "The URL should be of 200 characters at max"}
+            )
+
+        regex = re.compile("^https://[a-z]{2,3}[.]linkedin[.]com/.*$", re.I)
```

Comment:
I think it makes sense to check the URL for a linkedin domain name, but are you sure that they use a shortened or localized domain? 

We should at least take a look at the data we've already collected from users.

Reply:
Correct, We can take a look at the collected data and base this logic on that. I can verify that data on RC would that be enough? or would it be a good idea to take a look at a more realistic data set?